ci/jenkins/pipelines/prs/skuba-test-vmware.Jenkinsfile

```diff
@@ -95,17 +96,37 @@ pipeline {
             steps {
                 sh(script: 'make -f skuba/ci/Makefile pre_deployment', label: 'Pre Deployment')
                 sh(script: 'make -f skuba/ci/Makefile pr_checks', label: 'PR Checks')
+                sh(script: "pushd skuba; make -f Makefile install; popd", label: 'Build Skuba')
             }
         }
 
-        stage('Cluster Deployment') {
-            when {
-                expression { return shouldRun }
+        stage('Cluster Provisioning') {
+            steps {
+                sh(script: 'make -f skuba/ci/Makefile create_environment', label: 'Provision')
             }
+        }
+
+       stage('Run Pre Bootstrap Tests') {
```

Comment:
Do we really make use of those Pre/Post Bootstrap Tests? I mean, if we are not using them and we do not have now a current specific need or wish for them I'd just remove those steps as we are about to do in #963. We an always make our tests more complex later if we need it.

Reply:
yep, no reason but I wanted to sync the jobs first. It will be removed as part of 963 as you mention :+1: